### PR TITLE
fix: delete unused return args

### PR DESCRIPTION
**Pull request by lucming**
### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: pkg/koordlet/executor/resource_update_executor.go [hunk 1]
```diff
@@ -219,6 +219,8 @@ func (e *LeveledResourceUpdateExecutor) LeveledUpdateBatchByCache(resources [][]
 			}
 		}
 	}
+
+	updated = err == nil
```
Comment (zwzhang0107) [inline pkg/koordlet/executor/resource_update_executor.go:223]: i suggest we can remove the return value since the meaning is alreay fuzzy.

Comment (lucming) [reply pkg/koordlet/executor/resource_update_executor.go]: > i suggest we can remove the return value since the meaning is alreay fuzzy.

done

**Review by jasonliu747 — COMMENTED** [review]
/lgtm
@lucming 辛苦更新一下 PR 标题

**Pull request merged**